python/srv.py

```diff
@@ -25,23 +25,34 @@
 
 logging.basicConfig(level=logging.INFO)
 
+parser = argparse.ArgumentParser()
+parser.add_argument(
+    'database', help='path to database file, in-memory will be used if empty')
```

Comment:
Will do (but with `nargs='?'` :) ) - at first I didn't want to change it to avoid changing C++ code, but meh.